frontend/packages/db-structure/src/parser/drizzle/tableParser.ts

```diff
@@ -132,6 +133,95 @@ export const parsePgTableCall = (
   return table
 }
 
+/**
+ * Parse schema.table() call expression
+ */
+export const parseSchemaTableCall = (
```

Comment:
The main change is in `parseSchemaTableCall`.
It parses schemas like the one below:

```ts
export const users = authSchema.table('users', {
  id: serial('id').primaryKey(),
  email: varchar('email_address', { length: 255 })
}, (table) => ({
  emailIdx: index('email_idx').on(table.email)
}))
```
You can find the test code here:
https://github.com/liam-hq/liam/blob/10bad6faa11701b2efc8fdf4149ddd6120a407aa/frontend/packages/db-structure/src/parser/drizzle/index.test.ts#L737